tests/contrib/anthropic/test_anthropic.py

```diff
@@ -221,6 +221,44 @@ def test_anthropic_llm_sync_tools(anthropic, request_vcr):
         assert message is not None
 
 
+@pytest.mark.snapshot(
```

Comment:
Can we add corresponding tests to `test_anthropic_llmobs.py`?
1. Sync full tool use
2. Asnyc full tool use
3. Sync tool use with stream
4. Async tool use with stream helper

Reply:
done